andela/ah-backend-athena
# #163225414 ft-signup-email-confirmation-redirect

### What does this PR do?
This pull request enables one to be redirected to the frontend when one clicks the on the signup verification link.

### Description of tasks to be completed?
Add frontend domain name to the environment variables with key `FRONT_END_DOMAIN`
Write code to redirect to `/login` if the link is right
Write code to redirect to `/invalid_link` if the link is invalid


## How should this be tested?
Use https://ah-backend-athena-stagin-pr-39.herokuapp.com/ to create a user and validate him using the link sent it the email.

The link should redirect you to the login page

### Screen shots
<img width="922" alt="screen shot 2019-01-16 at 19 26 10" src="https://user-images.githubusercontent.com/10868919/51263072-a4c26b00-19c4-11e9-8914-62cb2283a8ad.png">

<img width="1086" alt="screen shot 2019-01-16 at 19 56 09" src="https://user-images.githubusercontent.com/10868919/51265040-d50c0880-19c8-11e9-9651-1caf6d431a48.png">
